src/main/java/bowling/view/InputView.java

```diff
@@ -0,0 +1,37 @@
+package bowling.view;
+
+import bowling.exception.InvalidNameOfPlayerException;
+
+import java.util.Scanner;
+
+public class InputView {
+    private static final Scanner SCANNER = new Scanner(System.in);
+    private static final String INPUT_NAME_OF_PLAYER = "플레이어 이름은(3 english letters)?: ";
+    private static final String INPUT_NUMBER_OF_FALLEN_PINS_IN_FRAME = "%s프레임 투구 : ";
+
+    private InputView() {
+    }
+
+    public static String inputNameOfPlayer() {
+        try {
+            System.out.println(INPUT_NAME_OF_PLAYER);
```

Comment:
```suggestion
    public static String inputNameOfPlayer() {
        try {
            System.out.print(INPUT_NAME_OF_PLAYER);
```

> java.util.NoSuchElementException: No line found 에러로 인해 프로그램이 실행되지 않더라구요. 결국 nextLine() 사용 전에 print()가 아닌 println()를 사용해서 해결하였습니다. 도메인 관련 질문은 아니지만 혹시 이 부분을 어떻게 요구사항의 화면처럼 구현할 수 있을지요?

저는 print 해도 잘되던데요? 신기하군요...

<img width="764" alt="스크린샷 2022-05-17 오전 10 44 04" src="https://user-images.githubusercontent.com/38595650/168710798-0bff75aa-c1d6-4b5c-85ca-ffd73d87b271.png">

Reply:
헉.. 제 컴퓨터만 그런 거였다니.... 미스터리하네요....